ui/src/components/pages/Login/index.tsx

```diff
@@ -26,23 +29,117 @@ export function Login() {
   }, [returnURL]);
   return (
     <div className="flex row loginPageContainer">
-      <div className="flex row logoContainer">
-        <div className="flex column ">
-          <span
+      <div className="flex column logoContainer" style={{ width: "100%" }}>
+        <div
+          className="flex column"
+          style={{ alignItems: "center", marginTop: "5rem" }}
+        >
+          <div
             style={{
               color: "#fff",
-              width: "82%",
               fontSize: "32px",
             }}
+            className="flex row"
           >
-            Unlock the power of data streaming with Numaflow!
-          </span>
+            Unlock the power of
```

Comment:
Can you update this to be one div? Just need to set a max-width on it to get the stacking you want. 

Also, convert px to rem